Lib/fontbakery/utils.py

```diff
@@ -96,6 +96,19 @@ def parse_version_string(fb, s):
                  " version numbers in '{}'".format(s))
 
 
+def parse_version_head(fonts):
+    """Return a family's version number. Ideally, each font in the
+    family should have the same version number. If not, return the highest
+    version number."""
+    versions = []
+    if isinstance(fonts, list):
+      for font in fonts:
+        versions.append(float(font['head'].fontRevision))
+    else:
+      versions.append(float(fonts['head'].fontRevision))
+    return max(versions)
```

Comment:
You really want font version numbers to be consistent across a family so returning the max is the safest bet. However, it would be cool to add this min/max in the future. We already have a check which sees if the version is consistent across a family. I can refactor this function and the check later on.